cookbooks/aws-parallelcluster-config/files/default/imds/imds-access.sh

```diff
@@ -35,10 +35,11 @@ EOF
 }
 
 function iptables_delete() {
-  local chain=$1
-  local destination=$2
-  local jump=$3
-  local user=$4
+  local $iptables_command=$1
```

Comment:
I think there is an error here. We shouldn't have the $ symbol on the left

Reply:
Thanks! https://github.com/aws/aws-parallelcluster-cookbook/pull/1464 fixes this bug